app_build_suite/build_steps/helm.py

```diff
@@ -297,9 +297,13 @@ def run(self, config: argparse.Namespace, _: Context) -> None:
             self._kubelinter_bin,
             "lint",
             config.chart_dir,
+            "--verbose",
         ]
-        if config.debug:
-            args.append("--verbose")
+
+        # adding `--verbose` to the args by default due to the kube-linter hidding
+        # some problems and thus skipping lintering some of the resources without informing
+        # the user.
```

Comment:
```suggestion
        # adding `--verbose` to the default args due to the kube-linter hiding
        # some problems and thus skipping linting some of the resources.
```

Some minor edits. WDYT? 

The linter is there to inform the user. So I think the last part is implied.

Reply:
sure! Thanks @rossf7